cmd/src/actions_exec.go

```diff
@@ -489,3 +531,28 @@ func diffStatDiagram(stat diff.Stat) string {
 	}
 	return color.GreenString(strings.Repeat("+", int(added))) + color.RedString(strings.Repeat("-", int(deleted)))
 }
+
+// askForConfirmation asks the user for confirmation. A user must type in "yes" or "no" and
+// then press enter. It has fuzzy matching, so "y", "Y", "yes", "YES", and "Yes" all count as
+// confirmations. If the input is not recognized, it will ask again. The function does not return
+// until it gets a valid response from the user.
+func askForConfirmation(s string) bool {
+	reader := bufio.NewReader(os.Stdin)
+
+	for {
+		fmt.Printf("%s [y/n]: ", s)
+
+		response, err := reader.ReadString('\n')
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		response = strings.ToLower(strings.TrimSpace(response))
+
+		if response == "y" || response == "yes" {
+			return true
+		} else if response == "n" || response == "no" {
```

Comment:
maybe everything else just defaults to no? That's what `terraform` does

Reply:
Sure. I just copy/pasted this function from somewhere, haven't looked into it that much yet (branch is still a draft)

Reply:
oops, I could swear I read it was not 🤦‍♂